### PR TITLE
Fixing post-preview organism

### PR DESCRIPTION
**Pull request by KimberlyMunoz**
- Post preview headline is now a link to the post
- Adds .external_link_text and .external_link_url for the ability to add an additional link to post preview
- Fixes a few spacing issues with post-preview organism.
- Resolves issue #1434

## Testing
- Create a browse filterable page and publish it locally with the "Amicus Brief" filter
- Create lots of document-detail pages under the browse-filterable page with "amicus brief" page types.
- Test~

## Review
- @kave 
- @jimmynotjim 
- @sebworks 
- @anselmbradford

### CODE REVIEW

File: cfgov/unprocessed/css/organisms/post-preview.less [hunk 1]
```diff
@@ -94,6 +99,10 @@
         display: block;
         width: 100%;
     }
+    &_title {
+        color: @black;
+        .u-link-child__hover(@pacific-80);
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/organisms/post-preview.less]: Could this be simplified by moving the link into the heading and using the color utility?

```
&_title a {
  .u-link__colors( @black, @pacific-80 );
}

File: cfgov/unprocessed/css/organisms/post-preview.less [hunk 2]
```diff
@@ -82,6 +82,11 @@
 */
 
 .o-post-preview {
+
+    + .o-post-preview {
+        margin-top: unit( @grid_gutter-width * 2 / @base-font-size-px, em);
```
Comment (jimmynotjim) [inline cfgov/unprocessed/css/organisms/post-preview.less]: Should the `@grid_gutter-width * 2` be wrapped in parens?

**Comment by jimmynotjim** [comment]
Works great locally, just a couple of questions.

File: cfgov/unprocessed/css/organisms/post-preview.less (hunk 2 above)
Comment (KimberlyMunoz) [inline cfgov/unprocessed/css/organisms/post-preview.less]: It renders to 60px. I think it's fine without.

File: cfgov/unprocessed/css/organisms/post-preview.less (hunk 1 above)
Comment (KimberlyMunoz) [inline cfgov/unprocessed/css/organisms/post-preview.less]: fixed in f75e3b1

File: cfgov/jinja2/v1/_includes/organisms/post-preview.html [hunk 3]
```diff
@@ -157,9 +161,9 @@ <h3 class="o-post-preview_title">
                     {{ tags.render(post.tags.names(), url, true, false, form_id) }}
                 {% endif %}
             </div>
-            {% if post.preview_link_text %}
-                <a href="{{ get_page_state_url({'request':request}, page) }}" class="jump-link jump-link__underline">
-                    {{ post.preview_link_text }}
+            {% if post.additional_link_text %}
```
Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/_includes/organisms/post-preview.html]: Possibly revert this change. I realized this is in the CMS already.

**Comment by KimberlyMunoz** [comment]
Fixed. I just noticed that in order to fix the issue with the links, the BEWDs will have to add a field here for the preview link url.

image

**Comment by KimberlyMunoz** [comment]
Hey @schaferjh!

After a little bit of chatting with the other FEWDs, we need some clarification on the hyperlink in the post-preview.

image

Originally we had the hyperlink link to the post, but I heard in a team working session that we wanted that to be an additional link for a certain use case and have the *title* hyperlinked to the post. Can you share the use case and clarify the spec for the hyperlink?

**Comment by KimberlyMunoz** [comment]
Jess responded with https://github.com/cfpb/cfgov-refresh/issues/1434#issuecomment-182555450 and the following


> right, that thing being called "preview_link" needs to be able to be a different url than the title
>
>"preview_link" is probably a bad name -- "additional_link" would be better etc
>
>and the vast majority of Post previews will not use the additional link -- the title will be the main link, as you see on the blog on beta today

File: cfgov/jinja2/v1/_includes/organisms/post-preview.html [hunk 4]
```diff
@@ -31,8 +31,8 @@
    settings.event_zip:          A string with the event zip.
 
    settings.comments_close_date: A datetime object marking the deadline of the comment period.
-   settings.link_text:           A string with the description text of an external link.
-   settings.link_url:            A string with the url of an external link.
+   settings.preview_link_text:   A string with the description text of an external link.
+   settings.preview_link_url: A string with the url of an external link.
```
Comment (sebworks) [inline cfgov/jinja2/v1/_includes/organisms/post-preview.html]: Does this need and extra space to be like the others?

**Comment by sebworks** [comment]
:+1:

File: cfgov/jinja2/v1/_includes/organisms/post-preview.html (hunk 4 above)
Comment (KimberlyMunoz) [inline cfgov/jinja2/v1/_includes/organisms/post-preview.html]: Thanks! Fixed in c573ed1

**Comment by jimmynotjim** [comment]
LGTM

**Comment by kurtw** [comment]
> Fixed. I just noticed that in order to fix the issue with the links, the BEWDs will have to add a field here for the preview link url.

What's the issue with links?

**Comment by kurtw** [comment]
I'll update the Post Preview organism to add a hyperlink.

**Comment by kurtw** [comment]
https://github.com/cfpb/cfgov-refresh/pull/1481
